Tests/govuk_ios/govuk_ios_unit_tests/Mocks/MockCoordinatorBuilder.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+import Foundation
+
+@testable import govuk_ios
+
+class MockCoordinatorBuilder: CoordinatorBuilder {
+    
+    var _stubbedOnboardingCoordinator: MockBaseCoordinator?
+    var _receivedOnboardingNavigationController: UINavigationController?
+    override func onboarding(navigationController: UINavigationController,
+                             dismissAction: @escaping () -> Void) -> BaseCoordinator {
+        _receivedOnboardingNavigationController = navigationController
+        return _stubbedOnboardingCoordinator ??
+        MockBaseCoordinator(
+            navigationController: navigationController
+        )
+    }
+    
+    var _stubbedTabCoordinator: MockBaseCoordinator?
+    var _receivedTabNavigationController: UINavigationController?
+    override func tab(navigationController: UINavigationController) -> BaseCoordinator {
+        _receivedTabNavigationController = navigationController
+        return _stubbedTabCoordinator ??
+        MockBaseCoordinator(
+            navigationController: navigationController
+        )
+    }
+    
+    var _stubbedLaunchCoordinator: MockBaseCoordinator?
+    var _receivedLaunchNavigationController: UINavigationController?
+    var _receivedLaunchCompletion: (() -> Void)?
+    override func launch(navigationController: UINavigationController,
+                         completion: @escaping () -> Void) -> BaseCoordinator {
+        _receivedLaunchNavigationController = navigationController
+        _receivedLaunchCompletion = completion
+        return _stubbedLaunchCoordinator ??
+        MockBaseCoordinator(
+            navigationController: navigationController
+        )
+    }
+    
+    var _stubbedRedCoordinator: MockBaseCoordinator?
```

Comment:
Do we still need these colour mocks?

Reply:
Looks like this is a legacy mock, I will remove